src/applications/disability-benefits/all-claims/pages/uploadPtsdForms.js

```diff
@@ -34,40 +36,23 @@ export const uiSchema = {
     parseResponse: (response, file) => ({
       name: file.name,
       confirmationCode: response.data.attributes.guid,
+      attachmentId: 'VA Form 21-781 - Statement in Support of Claim for PTSD',
```

Comment:
Is this supposed to be the attachment ID name, or the ID code? (example: `L027`)? Like what the 781a upload `attachmentId` ?